README.md

```diff
@@ -146,31 +158,40 @@ Mozilla Thunderbird | Supported | 115.7.0 |            |
 
 ### Graphics/design/CAD
 
-Name | Official | Tested | Compatible | Patched
------|----------|--------|------------|--------
-     |          |        |            |
+Name      | Official | Tested | Compatible | Patched
+----------|----------|--------|------------|--------
+Blender   | 2.92     |        |            | 🇵 4.1.1
+Paint.NET | 4.3.12   | 4.3.12 |            |
+
+[BlenderCompat (Fork of blender with Windows 7 compatability)](https://github.com/nalexandru/BlenderCompat/releases)
+[Paint.NET](https://archive.org/details/paintdotnet_v4_3_12)
 
 ### Developer tools/IDEs
 
-Name              | Official  | Tested   | Compatible | Patched
-------------------|-----------|----------|------------|----------
-Git               | Supported | 2.40.1   |            |
-GitKraken         | Dropped   | 9.11.0   |            |
-Insomnia          | Broken    | 2022.6.0 |            |
-VS Code/VS Codium | 1.70.3    | 1.70.3   |            | 🇵 1.86.0
-WakaTime          | Broken    |          |            |
+Name                 | Official       | Tested   | Compatible | Patched
+---------------------|----------------|----------|------------|----------
+Git                  | Supported      | 2.40.1         |            |
+GitKraken            | Dropped        | 9.11.0         |            |
+Insomnia             | Broken         | 2022.6.0       |            |
+VS Code & VS Codium  | 1.70.3         | 1.70.3         | 1.79.2     | 🇵 1.86.0
+Codium (VSCode fork) | ❗ 1.88.1.24104 | ❗ 1.88.1.24104 |            |
+WakaTime             | Broken         |                |            |
+Rust Compiler        | 1.78           |                |            |
+
+Notes: Rust can still compile for Windows 8.1 - 7, but the support is no longer a high priority and sucess building binaries made aren't guaranteed. As for Go, binaries created in Go only work on Windows 10 and above.
```

Comment:
This better fits in `Languages/frameworks` section above. I can't speak for Rust, but I know for a fact that binaries built with a version of Go compatible with Windows 7 (such as 1.21.4, but not 1.21.5 anymore) do run just fine on W7. I don't know if the compiler itself will run at all starting with 1.21.5.